src/js/components/DataView/stories/Simple.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { Box, Data } from 'grommet';
+
+import { DataView } from '../DataView';
+import { DATA } from '../../DataTable/stories/data';
+
+export const Simple = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Box fill align="center" justify="start" pad="large">
+    <Data
+      data={DATA}
+      views={[
+        { name: 'latest', sort: { property: 'date', direction: 'desc' } },
+        { name: 'behind', properties: { percent: { min: 0, max: 30 } } },
+      ]}
+    >
+      <DataView />
```

Comment:
If DataView is used on its own (not within toolbar) I'm wondering if we should provide a way to clear the selected view.

Reply:
Would the "Clear filters" control be enough?

Reply:
Yes